### PR TITLE
arch: rewrite all scripts based on ubuntu

### PR DESCRIPTION
**Pull request by gotenksIN**

### CODE REVIEW

**Comment by gotenksIN** [comment]
do not merge until its squashed/history is sanitized

File: arch/install_efistub.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Use this as a reference, update loader/initrd according to your kernel/microcode setup
+
+label='rendumOS (efistub)'
+loader='vmlinuz-linux'
+initrd='initramfs-linux.img'
+
+echo "Enter your boot disk device (e.g. /dev/sda)"
+read disk
+
+echo "Enter partition number for your boot partition"
```
Comment (akhilnarang) [inline arch/install_efistub.sh]: Maybe give an example here like above, so that people don't get confused

File: arch/install_efistub.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Use this as a reference, update loader/initrd according to your kernel/microcode setup
+
+label='rendumOS (efistub)'
```
Comment (akhilnarang) [inline arch/install_efistub.sh]: Read label from user? Arch users can just directly run it then, without editing the script

File: arch/setup.sh [hunk 3]
```diff
@@ -6,7 +6,7 @@ makepkg -si
 cd ..
 rm -rf yay
 
-sudo yay -Sy fortune-mod figlet zsh htop git ncdu nano base-devel wget curl aria2 inetutils bat ripgrep p7zip ookla-speedtest-bin
```
Comment (ksdfg) [inline arch/setup.sh:9]: why get rid of git and base-devel?

**Review by akhilnarang — CHANGES_REQUESTED** [review]

File: arch/install_efistub.sh (hunk 2 above)
Comment (gotenksIN) [reply arch/install_efistub.sh]: editing would be needed for kernel change or microcode additions if any
like if I am using linux-zen with amd-ucode I'd append -zen to loader and initrd and add another initrd=\amd-ucode.img before initramfs
I am unsure how do that on-the-fly (microcode part)

File: arch/setup.sh (hunk 3 above)
Comment (gotenksIN) [reply arch/setup.sh:9]: git would be needed for cloning repo and base-devel is a requirement for building yay before

Comment (gotenksIN) [reply arch/setup.sh:9]: I could probably add "minimum requirement: a user with sudo privileges and git and base-devel installed" somewhere to clarify maybe?

File: arch/install_efistub.sh (hunk 2 above)
Comment (akhilnarang) [reply arch/install_efistub.sh]: Yeah, I just meant for the label
(I'm assuming if random people use the script they'll be on stock kernel, if they're installing custom kernels they'll likely install efistub by themselves, I think)

Comment (gotenksIN) [reply arch/install_efistub.sh]: https://github.com/gotenksIN/scripts/pull/5/commits/238c115d87aa4ded50f3d38a91cd311cb6b010c1 this should be enough for that
now microcode handling seems like a bitch, I almost want to ifelse it out but that seems a bit too shitty

File: arch/setup.sh (hunk 3 above)
Comment (gotenksIN) [reply arch/setup.sh:9]: https://github.com/gotenksIN/scripts/commit/6cb501a5fc37765a6773181e61c3b8131f5fcc8d should fix it

**Comment by gotenksIN** [comment]
good to merge from my end

**Review by ksdfg — APPROVED** [review]

**Review by akhilnarang — APPROVED** [review]

**Pull request merged**